.github/workflows/content-sources-actions.yml

```diff
@@ -134,6 +149,10 @@ jobs:
           CLIENTS_PULP_SERVER: http://localhost:8080
           CLIENTS_PULP_USERNAME: admin
           CLIENTS_PULP_PASSWORD: password
+          CLIENTS_CANDLEPIN_SERVER: http://localhost:8181
```

Comment:
this should probably be  http://localhost:8181/candlepin

Reply:
thanks!